File: backend-example-java/src/main/java/com/example/backend/application/test/TestIocService2.java
package com.example.backend.application.test;

/**
 * 循环依赖测试类2
 */
//@Component
public class TestIocService2 {

    private final TestIocService1 service1;

    public TestIocService2(TestIocService1 service1) {
        this.service1 = service1;
    }
}

File: backend-example-dotnet/src/application/test/TestIocService2.cs
namespace BackendExample.Application.Test
{
    /// <summary>
    /// 循环依赖测试类2
    /// </summary>
    public class TestIocService2
    {
        private readonly TestIocService1 service1;

        public TestIocService2(TestIocService1 service1)
        {
            this.service1 = service1;
        }
    }
}
